packages/vuetify/src/labs/VNumberInput/VNumberInput.tsx

```diff
@@ -294,9 +297,9 @@ export const VNumberInput = genericComponent<VNumberInputSlots>()({
 
               { incrementControlNode() }
             </div>
-          ) : (!props.reverse
```

Comment:
Is this change just a code format? Or was there any logically concrete reason behind it?

Reply:
Only code format to remove the exclamation mark. I believe it's easier to read especially when there is the same condition with another case.